tests/testTriggers.go

```diff
@@ -642,51 +709,100 @@ func rangeStructer(args ...interface{}) []interface{} {
 	return out
 }
 
-var htmlTemplate = `
-<!DOCTYPE html>
+var htmlTemplate = `<!DOCTYPE html>
 <html>
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js"></script>
 <style>
 table {
   border-collapse: collapse;
-  width: 100%;
 }
 th {
    background-color: #0ca1f0;
    text-align: center;
    padding: 3px;
 }
 td {
-  text-align: left;
+  text-align: center;
   padding: 3px;
 }
+
 tbody tr:nth-child(even) {
-	background-color: #bac5ca;
+  background-color: #bac5ca;
 }
 tbody tr:last-child {
   background-color: #79ab78;
 }
+@media only screen and (max-width: 500px) {
+	.wrapper table {
+		width: 100% !important;
+	}
+
+	.wrapper .column {
+		// make the column full width on small screens and allow stacking
+		width: 100% !important;
+		display: block !important;
+	}
+}
 </style>
 </head>
 <body>
 <h1>Torpedo Longevity Report</h1>
 <hr/>
+<h3>SetUp Details</h3>
+<p><b>Master IP:</b> {{.MasterIP}}</p>
+<table id="pxtable" border=1 width: 50% >
+<tr>
+   <td align="center"><h4>PX Node IP </h4></td>
+   <td align="center"><h4>PX Version </h4></td>
+   <td align="center"><h4>PX Status </h4></td>
+ </tr>
+{{range .NodeInfo}}<tr>
+{{range rangeStruct .}} <td>{{.}}</td>
+{{end}}</tr>
+{{end}}
+</table>
+<hr/>
+<h3>Running Event Details</h3>
+<table border=1 width: 50%>
+<tr>
+   <td align="center"><h4>Trigget Name </h4></td>
+   <td align="center"><h4>Interval </h4></td>
+ </tr>
+{{range .TriggersInfo}}<tr>
+{{range rangeStruct .}} <td>{{.}}</td>
+{{end}}</tr>
+{{end}}
+</table>
+<hr/>
 <h3>Event Details</h3>
-<table border=1>
+<table border=1 width: 100%>
 <tr>
-   <td align="center"><h4>Event </h4></td>
+   <td class="wrapper" width="600" align="center"><h4>Event </h4></td>
    <td align="center"><h4>Start Time </h4></td>
    <td align="center"><h4>End Time </h4></td>
    <td align="center"><h4>Errors </h4></td>
  </tr>
-{{range .Records}}<tr>
-{{range rangeStruct .}}	<td>{{.}}</td>
+{{range .EmailRecords.Records}}<tr>
+{{range rangeStruct .}} <td>{{.}}</td>
 {{end}}</tr>
 {{end}}
 </table>
+<script>
+$('#pxtable tr td').each(function(){
+  var cellValue = $(this).html();
+  
+    if (cellValue === "OPERATIONAL") {
+      $(this).css('background-color','green');
```

Comment:
Is this color coding working? In the sample screenshot attached, the background color is not seen.

Reply:
Currently, it is not working, but left this change to work on future